docs/source/app_development/examples.rst

```diff
@@ -1,8 +1,8 @@
-.. _develop-apps:example-qe_bands:
+.. _develop-apps:examples:
 
-****************************************************
-Build an app - AiiDA Quantum ESPRESSO bands workflow
-****************************************************
+************
+Example apps
+************
 
 Here we will have an example of how to build an AiiDAlab app that utilizes `AiiDA <https://www.aiida.net>`__ with the `AiiDA-Quantum ESPRESSO plugin <https://aiida-quantumespresso.readthedocs.io/>`__ to calculate the band structure for a compound.
```

Comment:
I think writing a QE app (even a simple one) as an example is total overkill. I would suggest that we change this to describe something much simpler.

Reply:
What if we just start with a set of links to example apps?
This would little effort and probably very helpful?

That was the thought behind the change of the title - I'll change the text as well